docs/reference/samples/observability.filters-pipelines-unchanged.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: "observability.openshift.io/v1"
+kind: ClusterLogForwarder
```

Comment:
Are these intended to be working examples?  If so then they should at minimum have a name.  Namespace is implied with  the create command

Reply:
Understood.  I keep flipping back and forth between samples and snippets.   Will get this straightened out today.

Reply:
I think we are going to need both types of examples/samples here.   I've included two input spec examples, but not sure its necessary to show outputs for those.    I still think these are helpful to have somewhere, if we want to only provide full examples.   There are complete examples where I think it makes sense.  Also, I have many more examples.